src/seth/libexec/seth/seth-send

```diff
@@ -24,7 +24,7 @@ set -e
 [[ $ETH_FROM ]] ||
 seth --fail "${0##*/}: error: \`ETH_FROM' not set; send from which account?"
 [[ $ETH_GAS ]] ||
-echo >&2 "${0##*/}: warning: \`ETH_GAS' not set; using default gas amount"
+echo >&2 "${0##*/}: warning: \`ETH_GAS' not set; using estimated gas amount"
```

Comment:
Thoughts on removing this warning? I think this is typically the expected behavior when sending transactions that don’t specify a gas limit, so a warning seems unnecessary

Reply:
Yeah, I agree. Will change now